src/AltinnCore/Testdata/User/1/partylist.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "SSN": "01017512345",
+    "PartyId": "1",
```

Comment:
should partyId be updated to be more realistic wrt. what we use today?

Reply:
Ya, but i guess we could get things work first and i suppose there is an issue that is about testdata